static/src/stylesheets/module/atoms/_explainers.scss

```diff
@@ -0,0 +1,194 @@
+@mixin clean($element) {
+    #{$element}.clean {
+        @content;
+    }
+}
+
+@include clean(details) {
+    > summary {
+        outline: none;
+
+        &::-webkit-details-marker {
+            display: none;
+        }
+    }
+}
+
+.explainer-snippet {
+    position: relative;
+    padding: $gs-baseline / 3 $gs-gutter / 4 $gs-baseline / 2;
+    margin: $gs-baseline * 4 / 3 0 $gs-baseline * 3;
+
+    font-family: $f-sans-serif-text;
+
+    &[open] .explainer-snippet__handle > :nth-child(1),
+    &:not([open]) .explainer-snippet__handle > :nth-child(2) {
+        display: none;
+    }
+}
+
+.explainer-snippet__header {
+    margin: 0 0 $gs-baseline * 4 / 3;
+    font-weight: bold;
+}
+
+.explainer-snippet__label {
+    @include font-size(14, 18);
+    display: block;
+}
+
+.explainer-snippet__headline {
+    @include font-size(20, 22);
+}
+
+.explainer-snippet__handle {
+    position: absolute;
+    bottom: 0;
+    transform: translate(0, 50%);
+    border: 0;
+    padding: 0 15px 0 7px;
+
+    > span,
+    .inline-icon {
+        display: inline-flex;
+        align-items: center;
+    }
+
+    svg {
+        width: 20px;
+        height: 20px;
+        margin-right: $gs-gutter / 2;
+    }
+}
+
+.explainer-snippet__body {
+    > p {
+        @include faux-bullet-point;
+
+        &::before {
+            margin-right: $gs-gutter / 4;
+        }
+    }
+}
+
+.explainer-snippet__footer {
+    @include font-size(13, 16);
+
+    display: flex;
+    justify-content: flex-end;
+}
+
+.explainer-snippet__feedback {
+    display: flex;
+    align-items: center;
+
+    .button {
+        border: 0;
+        margin: 0 0 0 $gs-gutter / 4;
+        padding: 0;
+        width: 28px;
+        height: 28px;
+
+        &[value='dislike'] {
+            transform: rotate(180deg);
+        }
+    }
+
+    .button,
+    .inline-icon {
+        display: inline-flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+    svg {
+        width: 16px;
+        height: 16px;
+    }
+
+    path {
+        fill: currentColor;
+    }
+}
+
+/** THEMING */
+
+.explainer-snippet--light {
+    background: #f1f1f1;
```

Comment:
#f1f1f1 is part of the colour palette for our Analysis tone. It's used as bg colour for the H1 (e.g. https://www.theguardian.com/world/2017/may/07/emmanuel-macron-elected-french-president-what-happens-now) and as bg colour for Analysis-tone rich links (e.g. bottom left column on https://www.theguardian.com/world/2017/may/17/macron-austrian-minister-sebastian-kurz-french-election).